exporter/otlp-logs/lib/opentelemetry/exporter/otlp/version.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Exporter
+    module OTLP
+      ## Current OpenTelemetry OTLP logs exporter version
+      VERSION = '0.1.0'
```

Comment:
Doesn't this namespacing clash with this existing exporter: https://github.com/open-telemetry/opentelemetry-ruby/blob/main/exporter/otlp/lib/opentelemetry/exporter/otlp/version.rb? I believe this has been replaced by the otlp-http exporter, but we should nevertheless think about the namespace for this (and the rest of our exporters). We currently have `OpenTelemetry::Exporter::OTLP` (deprecated), `OpenTelemetry::Exporter::OTLP::HTTP`, `OpenTelemetry::Exporter::OTLP::GRPC`, and `OpenTelemetry::Exporter::OTLP::Metrics`. In the end, I think we want the trace, metrics, and logs HTTP exporters all in `OpenTelemetry::Exporter::OTLP::HTTP`, but perhaps for the meantime this can go in `OpenTelemetry::Exporter::OTLP::Logs`, and we will make the move to HTTP when it's stable?

Reply:
Great catch, thank you! That makes a lot of sense with a report I received about the wrong version for an OTLP exporter when it was installed from this test branch 😅 

I'll make the fix.

Reply:
Updated in a53b6e371613112b291bac2100741d80bf64844a